src/components/donation/Steps/Submit/DAFCheckout/ManualDonation.tsx

```diff
@@ -14,7 +14,7 @@ export default function ManualDonation(props: DafCheckoutStep) {
         To complete this donation, please email your provider with the following
         information:
       </p>
-      <div className="grid rounded bg-gray-l5 dark:bg-navy-d3 p-3 text-sm leading-relaxed my-4">
+      <div className="grid rounded bg-gray-l4 dark:bg-navy-d3 p-3 text-sm leading-relaxed my-4">
```

Comment:
`bg-gray-l5` has blueish hue, `-l4` is more neutral and would blend to any accent